core/chains/evm/forwarders/orm.go

```diff
@@ -66,3 +69,47 @@ func (o *orm) FindForwarders(offset, limit int) (fwds []Forwarder, count int, er
 	}
 	return
 }
+
+// FindForwardersByChain returns all forwarder addresses for a chain.
+func (o *orm) FindForwardersByChain(evmChainId utils.Big) (fwds []Forwarder, err error) {
+	sql := `SELECT * FROM evm_forwarders where evm_chain_id = $1 ORDER BY created_at DESC, id DESC`
+	if err = o.q.Select(&fwds, sql, evmChainId); err != nil {
+		return
+	}
+	return
+}
+
+func (o *orm) FindForwardersInListByChain(evmChainId utils.Big, addrs []common.Address) ([]Forwarder, error) {
+	var fwdrs []Forwarder
+
+	arg := map[string]interface{}{
+		"addresses": addrs,
+		"chainid":   evmChainId,
+	}
+
+	query, args, err := sqlx.Named(`
+		SELECT * FROM evm_forwarders 
+		WHERE evm_chain_id = :chainid
+		AND address IN (:addresses)
+		ORDER BY created_at DESC, id DESC`,
```

Comment:
I think it would be reasonable to combine the `address` and `evm_chain_id` indexes since you would never query for the address without also specifying the chain ID. Also I noticed that there is a duplicate index on `address`, one of these should be removed.

You can probably replace `evm_forwarders_address_key`, `idx_forwarders_evm_chain_id` and `idx_forwarders_evm_address` with just one like so:

`CREATE UNIQUE INDEX idx_evm_chain_id_address ON evm_forwarders (evm_chain_id, address)`

This is going to give you a slight query speedup and reduce index size significantly.